charts/kubeit-rabbitmq-chart/values.yaml

```diff
@@ -0,0 +1,145 @@
+# Default values for RabbitMQ Helm chart.
+---
+tenantName: tenant2
+component: rabbitmq
+rabbitmqCluster:
+  metadata:
+  # name of the RabbitMQ cluster
+    name: testmq-tenant
+  spec:
+    replicas: 1
+    resources:
+      requests:
+        cpu: 500m
+        memory: 500Mi
+      limits:
+        memory: 800Mi
+    containers:
+      volumeMount:
+        name: persistence-testmq
+    initContainers:
+      volumeMount:
+        name: persistence-testmq
+
+  additionalConfig: |
+    log.console.level = info
+    channel_max = 1700
+    default_user_tags.administrator = true
+    vm_memory_high_watermark.absolute = 4Gi
+
+# it should be optionally to use external Azure File Share
+# it should support both: AKS build-in storage class and Azure File Share and external Azure File Share
+persistentVolume:
+  # Note: if external is set to true, and want to use External Azure File Share it must be set to file.csi.azure.com
+  # Set to false to use AKS built-in storage classes
+  external: false
+  # set the storage class name
+  storageClassName: azurefile-csi
+  capacity:
+  # configure PVC size
+    storage: 2Gi
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  mountOptions:
+    - dir_mode=0777
+    - file_mode=0777
+    - uid=0
+    - gid=0
+    - mfsymlinks
+    - cache=strict
+    - nosharesock
+    - nobrl
+  csi:
+    driver: file.csi.azure.com
+    volumeHandle: rabbitmq-tenant
+    volumeAttributes:
+      # name of Azure File Share
+      shareName: rabbitmq
```

Comment:
is the shareName link to Azure Storage account share?
if not, how to point to external Azure Storage account?

Reply:
It is the name of the file share in the storage account. This approach is used to point to the external Azure Storage account. If you are using the in built storage classes it will auto create the file share

Reply:
To point to external azure SA use: `nodeStageSecretRef` in PV

Reply:
Could you describe it in readme.md?